stac_api/runtime/src/config.py

```diff
@@ -74,6 +74,7 @@ def load_postgres_settings(self) -> "Settings":
                 postgres_user=secret["username"],
                 postgres_pass=secret["password"],
                 postgres_port=secret["port"],
+                use_api_hydrate=True,
```

Comment:
let's wait on turning this on and also make it configurable from deployment with `VEDA_RASTER_API_HYDRATE`. The hydrate feature is probably safe to turn on but it does change how pgstac stores and retrieves records enough that we probably want to test it out alone. I think the hydrate method also uses the collection collection item assets property and we should make sure that it doesn't contradict our current usage.